asserts/database.go

```diff
@@ -207,6 +218,15 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 		}
 	}
 
+	otherPredefinedBackstore := NewMemoryBackstore()
+
+	for _, a := range cfg.OtherPredefined {
+		err := otherPredefinedBackstore.Put(a.Type(), a)
+		if err != nil {
+			return nil, fmt.Errorf("error loading for use predefined assertion %v", a.Ref())
```

Comment:
The wording in the error message is a bit weird for me. How about `fmt.Errorf("cannot add predefined assertion %v: %s", a.Ref(), err)`